lib/wovnrb/services/html_converter.rb

```diff
@@ -142,7 +142,7 @@ def insert_hreflang_tags
     # Remove wovn snippet code from dom
     def strip_snippet
       @dom.xpath('//script').each do |script_node|
-        script_node.remove if script_node['src'] && script_node['src'] =~ /^\/\/j.(dev-)?wovn.io(:3000)?\//
+        script_node.remove if script_node['src'] && !script_node['data-wovnio'].blank? || [@store.settings['api_url_versioned_widget'], 'j.wovn.io'].any? { |url| script_node['src'].include? url }
```

Comment:
I think the following is better.
```
(script_node['src'] && [@store.settings['api_url_versioned_widget'], 'j.wovn.io'].any? { |url| script_node['src'].include? url } ) || !script_node['data-wovnio'].blank?
```

I mean the following condition.
`src` includes `'https://wovn.global.ssl.fastly.net/widget'` or 'j.wovn.io'
or
script has `data-wovnio`attr

And you can use `present?` as `!blank?`.

Reply:
Good idea, thanks.